### PR TITLE
Switch to use class-claim for dynamic db provisioning

### PR DESCRIPTION
**Pull request by trisberg**

### CODE REVIEW

File: csharp-rest-service/DATABASE.md [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
 
-[tap]
+Using the `config/workload.yaml` it is possible to build, test and deploy this application onto a
+Kubernetes cluster that is provisioned with Tanzu Application Platform (https://tanzu.vmware.com/application-platform).
+
+As with the local deployment a PostgreSQL instance needs to be available at the cluster.
+
+1. App Operator Tasks
+
+   - Create the service `ClassClaim` to be consumed by your workload that references your postgres instance:
```
Comment (ccheetham) [inline csharp-rest-service/DATABASE.md:20]:
```suggestion
   - Create the service `ClassClaim` to be consumed by your workload that references your PostgreSQL instance:

```

**Review by ccheetham — APPROVED** [review]
Minor nit with "postgresl/PostgreSQL" .

Otherwise looks good to me.

**Review by TimHess — APPROVED** [review]
LGTM

**Review by ccheetham — APPROVED** [review]
LGTM

**Pull request merged**